app/src/main/java/com/quran/labs/androidquran/util/QuranFileUtils.kt

```diff
@@ -195,6 +220,25 @@ class QuranFileUtils @Inject constructor(
         makeDirectory(getQuranAyahDatabaseDirectory(context))
   }
 
+  @WorkerThread
+  fun copyFromAssets(assetsFileName: String, filename: String, destination: String) {
+    val assets = appContext.assets
+    assets.open(assetsFileName)
+        .source()
+        .use { source ->
+          File(destination, filename).sink()
+              .buffer()
+              .use { destination -> destination.writeAll(source) }
+        }
+
+    if (filename.endsWith(".zip")) {
+      val zipFile = destination + File.separator + filename
+      ZipUtils.unzipFile(zipFile, destination, filename, null)
+      // delete the zip file, since there's no need to have it twice
+      File(zipFile).delete()
+    }
+  }
+
```

Comment:
`CopyDatabaseUtil` has similar code but will be removed in a future PR in sha' Allah since it's not used at the present.